builtin/game/misc.lua

```diff
@@ -166,6 +166,18 @@ function core.setting_get_pos(name)
 	return core.string_to_pos(value)
 end
 
+function core.get_world_limits()
+	local absolute_limit = 31000
```

Comment:
This isn't accurate.  The real limits vary in each dimension since they're rounded to the nearest multiple of the block size (16), but they're closer to 31912.  If this isn't correct mods will stop working correctly near the world edge, which is undesirable because some people try to go there as a challenge, and several have made it.